tests/debugger/do_backtrace.expected

```diff
@@ -1,20 +1,32 @@
 Connecting to: localhost:5001
 Stopped at tests/debugger/do_backtrace.js:15
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:28
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:37
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:40
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:32 (in test() at line:30, col:1)
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
-(jerry-debugger) Frame 0: tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
+(jerry-debugger) Command: next
+Stopped at tests/debugger/do_backtrace.js:28
+(jerry-debugger) Command: n
+Stopped at tests/debugger/do_backtrace.js:37
+(jerry-debugger) Command: next
+Stopped at tests/debugger/do_backtrace.js:40
+(jerry-debugger) Command: step
+Stopped at tests/debugger/do_backtrace.js:32 (in test() at line:30, col:1)
+(jerry-debugger) Command: next
+Stopped at tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
+(jerry-debugger) Command: s
+Stopped at tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
+(jerry-debugger) Command: bt
+Frame 0: tests/debugger/do_backtrace.js:23 (in foo() at line:21, col:1)
 Frame 1: tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
 Frame 2: tests/debugger/do_backtrace.js:40
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:24 (in foo() at line:21, col:1)
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:25 (in foo() at line:21, col:1)
-(jerry-debugger) Stopped at tests/debugger/do_backtrace.js:18 (in f4() at line:17, col:1)
-(jerry-debugger) Frame 0: tests/debugger/do_backtrace.js:18 (in f4() at line:17, col:1)
+(jerry-debugger) Command: n
+Stopped at tests/debugger/do_backtrace.js:24 (in foo() at line:21, col:1)
+(jerry-debugger) Command: n
+Stopped at tests/debugger/do_backtrace.js:25 (in foo() at line:21, col:1)
+(jerry-debugger) Command: s
+Stopped at tests/debugger/do_backtrace.js:18 (in f4() at line:17, col:1)
+(jerry-debugger) Command: backtrace
+Frame 0: tests/debugger/do_backtrace.js:18 (in f4() at line:17, col:1)
 Frame 1: tests/debugger/do_backtrace.js:25 (in foo() at line:21, col:1)
 Frame 2: tests/debugger/do_backtrace.js:33 (in test() at line:30, col:1)
 Frame 3: tests/debugger/do_backtrace.js:40
-(jerry-debugger) Press enter to stop JavaScript execution.
+(jerry-debugger) Command: c
+Press enter to stop JavaScript execution.
```

Comment:
These "Press enter to stop JavaScript execution." should be removed in interactive execution.